docs/VnVPlan/VnVPlan.tex

```diff
@@ -1237,6 +1238,17 @@ \subsubsection{Usability and Humanity Requirements}
 
 \end{enumerate}
 
+\subsubsection{Performance Requirements}
+\begin{enumerate}
+	\item \textbf{NFRT-PR1-1} % For PR1
+
+	      Type: Dynamic, Manual
+
+	      How test will be performed: The tester will additionally add a timer to every test to check if
+	      every user function responds within 0.5 seconds.
```

Comment:
Maybe add a reference to the section for which tests you are referring to like Leon did

Reply:
Where did Leon reference something, just for easy comparison

Reply:
https://github.com/arkinmodi/project-sayyara/blob/c3494833d6572dcc355151715fa9320085f57f8e/docs/VnVPlan/VnVPlan.tex#L1421-L1427